repository/rabbitmq/operator/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kudo.dev/v1beta1
+name: "rabbitmq"
+version: "0.1.0"
```

Comment:
@harryge00 can you change the version to `operatorVersion`? CI is failing because of that as we moved to KUDO `0.10.0`
and add the `appVersion` also which would be `3.8` in this case